app/Http/Controllers/V2/Nurseries/ViewNurseryWithFormController.php

```diff
@@ -7,13 +7,18 @@
 use App\Models\V2\Forms\Form;
 use App\Models\V2\Nurseries\Nursery;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\App;
 
 class ViewNurseryWithFormController extends Controller
 {
     public function __invoke(Request $request, Nursery $nursery): NurseyWithSchemaResource
     {
         $this->authorize('read', $nursery);
 
+        if ($request->query('lang')) {
+            App::setLocale($request->query('lang'));
```

Comment:
This seems like a pattern we want more generally applied. I don't know what the idiomatic pattern would be in Laravel... maybe a concern that gets mixed in at the base `Controller` level?

Reply:
I created a middleware that would handle this repetitive task.
you would just need to apply `->middleware('i18n')` in the end of the route.
This common pattern was repeated in many places.